apps/deploy-web/src/components/new-deployment/ManifestEdit.tsx

```diff
@@ -119,6 +123,13 @@ export const ManifestEdit: React.FunctionComponent<Props> = ({ editedManifest, s
     }
   };
 
+  useEffect(() => {
```

Comment:
let's do it the same way as it's done in the code above on L92 with `useWhen`, also let's move these two hooks close together